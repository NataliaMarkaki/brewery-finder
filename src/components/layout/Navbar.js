import React from 'react';

const Navbar = ({ title = 'Brewery Finder', icon = 'fas fa-beer' }) => (
  <nav className='navbar bg-primary'>
    <h1>
      <i className={icon} /> {title}
    </h1>
  </nav>
);

export default Navbar;
