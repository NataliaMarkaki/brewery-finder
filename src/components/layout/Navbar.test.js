import React from 'react';
import { render } from '@testing-library/react';
import Navbar from './Navbar';

test('renders app', () => {
  const { baseElement } = render(<Navbar />);
  expect(baseElement).toMatchSnapshot();
});
