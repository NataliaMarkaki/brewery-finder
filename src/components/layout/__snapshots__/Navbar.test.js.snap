// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<body>
  <div>
    <nav
      class="navbar bg-primary"
    >
      <h1>
        <i
          class="fas fa-beer"
        />
         
        Brewery Finder
      </h1>
    </nav>
  </div>
</body>
`;
